manifest.json

```diff
@@ -1,39 +1,92 @@
 {
-  "manifest_version": 3,
-  "name": "Hide Slack Activity",
-  "version": "0.0.5",
-  "description": "This hides the activity notification badge icon and the more unreads buttons on the Slack website.",
-   "permissions": ["scripting"],
-  "icons": {
-    "16": "images/slack-emoji-16.png",
-    "32": "images/slack-emoji-32.png",
-    "48": "images/slack-emoji-48.png",
-    "128": "images/slack-emoji-128.png"
-},
-"background": {
-    "service_worker": "scripts/background.js"
-},
-"web_accessible_resources": [
-  {
-    "resources": ["css/*.css"],
-    "extension_ids": [
-      "lohigdpnlkaabdijopklelgffjopnghf"
-    ]
+    "manifest_version": 3,
+    "name": "Ghost Glassdoor",
+    "version": "0.0.1",
+    "description": "This allows users to access Glassdoor without logging in.",
+    "permissions": ["scripting"],
+    "icons": {
+        "16": "images/detective-emoji-16.png",
+        "32": "images/detective-emoji-32.png",
+        "48": "images/detective-emoji-48.png",
+        "128": "images/detective-emoji-128.png"
+    },
+    "background": {},
+    "web_accessible_resources": [
+        {
+            "resources": ["css/*.css"],
+            "extension_ids": ["lohigdpnlkaabdijopklelgffjopnghf"]
+        }
+    ],
+    "content_scripts": [
+        {
+            "matches": [
+                "*://*.glassdoor.com/*",
```

Comment:
A list of country level domains for glassdoor. This was copied from another website so there are duplicates.